vue.config.js

```diff
@@ -2,7 +2,17 @@ module.exports = {
   transpileDependencies: [
     'vuetify',
   ],
+
   devServer: {
     disableHostCheck: true,
   },
+
+  pwa: {
```

Comment:
enable pwa support so we can "add to homescreen" on phones, create desktop apps from browsers, and install in Windows 10.